test/e2e/egressqos.go

```diff
@@ -321,5 +328,25 @@ func pingAndCheckDSCP(f *framework.Framework, srcPod, dstPod1, dstPod1IP, dstPod
 	framework.ExpectNoError(err, "Failed to ping dst pod")
 	err = tcpDumpSync.Wait()
 	framework.ExpectNoError(err, "Failed to detect ping with correct DSCP on pod")
+}
 
+func ensureEgressQoSAppliedCorrectly(f *framework.Framework) {
+	nodes, err := e2enode.GetReadySchedulableNodes(context.TODO(), f.ClientSet)
+	framework.ExpectNoError(err)
+	gomega.Eventually(func() bool {
+		output, err := e2ekubectl.RunKubectl(f.Namespace.Name, "get", "egressqos", "default", "-o", "json")
+		if err != nil {
+			framework.Failf("could not get the egressqos default in namespace: %s", f.Namespace.Name)
+		}
+		for _, node := range nodes.Items {
+			nodeZone, ok := node.Annotations[ovnNodeZoneNameAnnotation]
+			if !ok {
+				framework.Failf("zone for the node %s not set in the annotation %s", node.Name, ovnNodeZoneNameAnnotation)
+			}
+			if !strings.Contains(output, fmt.Sprintf("%s: EgressQoS Rules applied", nodeZone)) {
+				return false
+			}
+		}
```

Comment:
you don't really need to check per-node status, as status manager will only update status.status with success when all nodes report success. Something like this https://github.com/ovn-org/ovn-kubernetes/pull/4035/files#diff-21d1ea5f18ba684bac6d206656f3386f5c30ed7df5a9de88d811628caf1b64cbR56-R65

Reply:
yes @npinaeva, just noticed `applyEmptyOrFailed` parameter in status manager takes care of it. 
skipping check on per-node status now.